src/main/java/app/screens/servants/ServantsController.java

```diff
@@ -0,0 +1,173 @@
+
+    @FXML
+    private Button buttonAddNew, buttonEdit, buttonSingleReport, buttonGeneralReport;
+
+    private String addServantPath = "/modals.addServant/AddServantModal.fxml";
+
+    @FXML
+    private void initialize() {
+        fill();
+    }
+
+    private ObservableList<PublicServant> loadValues() {
+        publicServants.add(new PublicServant("SC3528874", "Thomas Shelby"));
+        publicServants.add(new PublicServant("SC6248833", "Grace Shelby"));
+        publicServants.add(new PublicServant("SC3990183", "Arthur Shelby"));
+        publicServants.add(new PublicServant("SC7710223", "John Shelby"));
+        publicServants.add(new PublicServant("SC9044345", "Michael Gray"));
+        publicServants.add(new PublicServant("SC2566811", "Polly Gray"));
+        publicServants.add(new PublicServant("SC9134104", "Lizzie Stark"));
+        publicServants.add(new PublicServant("SC3404597", "Ada Thorne"));
+
+        return publicServants;
+    }
+
+    @FXML
+    private void fill() {
+        tableColumnRecord.setCellValueFactory(new PropertyValueFactory<>("id"));
+        tableColumnName.setCellValueFactory(new PropertyValueFactory<>("name"));
+
+        tableViewServants.setItems(loadValues());
+        tableViewServants.setPlaceholder(new Label("No Servants registered"));
+
+    }
+
+    @FXML
+    public void buttonAddNew_click() throws IOException {
+        Stage dialog = new Stage();
+        FXMLLoader loader = new FXMLLoader(getClass().getResource(addServantPath));
+        Parent root = loader.load();
+
+        AddServantController controller = loader.getController();
+        controller.setParentController(this);
+        dialog.setTitle("Add servant");
+        dialog.setScene(new Scene(root));
+        dialog.initModality(Modality.APPLICATION_MODAL);
+        dialog.setResizable(false);
+        dialog.showAndWait();
+    }
+
+    @FXML
+    public void buttonEdit_click() throws IOException {
+        PublicServant servant = tableViewServants.getSelectionModel().getSelectedItem();
+
+        if (servant != null) {
+            FXMLLoader loader = new FXMLLoader(getClass().getResource(addServantPath));
+            Parent root = loader.load();
+
+            AddServantController controller = loader.getController();
+            controller.setServant(servant);
+            controller.setParentController(this);
+
+            Stage dialog = new Stage();
+            dialog.setTitle("Edit servant");
+            dialog.setScene(new Scene(root));
+            dialog.initModality(Modality.APPLICATION_MODAL);
+            dialog.setResizable(false);
+            dialog.showAndWait();
+        }
+    }
+
+    @FXML
+    public void buttonSingleReport_click() throws IOException {
+        PublicServant servant = tableViewServants.getSelectionModel().getSelectedItem();
+
+        if (servant != null) {
+            FXMLLoader loader = new FXMLLoader(getClass().getResource("/screens.singleWorkloadReport/SingleReport.fxml"));
```

Comment:
I think that specify the path with dot character doens't work.

Reply:
It does work, that was the only way it would work